src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -49,18 +55,20 @@ public BtConnectorHelper() {
         this.context = jxcore.activity.getBaseContext();
     }
 
-
-    public BTConnector.WifiBtStatus Start(String peerName,int port){
+    public synchronized BTConnector.WifiBtStatus Start(String peerName,int port){
```

Comment:
:+1: 
BTW, we really need an atomic guard here so we can prevent someone accidentally calling startBroadcast twice without waiting for the callback. We should get an error on the second startBroadcast. But we can fix this later when I add a test for it.